pkg/util/csi_proxy_v1_windows.go

```diff
@@ -0,0 +1,93 @@
+//go:build windows
+// +build windows
+
+/*
+Copyright 2021 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"context"
+	"fmt"
+
+	volumeapi "github.com/kubernetes-csi/csi-proxy/client/api/volume/v1"
+	volumeclient "github.com/kubernetes-csi/csi-proxy/client/groups/volume/v1"
+)
+
+// CSIProxyV1 is the CSI Proxy implementation that uses the v1 API
+type CSIProxyV1 struct {
+	VolumeClient *volumeclient.Client
+}
+
+// check that CSIProxyV1 implements CSIProxy
+var _ CSIProxy = &CSIProxyV1{}
+
+func NewCSIProxyV1() (*CSIProxyV1, error) {
+	volumeClient, err := volumeclient.NewClient()
+	if err != nil {
+		return nil, err
+	}
+	return &CSIProxyV1{
+		VolumeClient: volumeClient,
+	}, nil
+}
+
+// GetAPIVersions returns the versions of the client APIs.
+func (proxy *CSIProxyV1) GetAPIVersions() string {
+	return fmt.Sprintf(
+		"API Versions Volume: %s",
```

Comment:
what volume mean here?

Reply:
it's the version of the Volume api group, this is what I also did in PD CSI to check the versions of the APIs that were used

Reply:
oh, maybe use CSI_PROXY API version?

Reply:
it's printed with the prefix CSI Proxy here: 

```
func NewCSIProxy() (CSIProxy, error) {
	csiProxyV1, err := NewCSIProxyV1()
	if err == nil {
		klog.V(2).Infof("using CSIProxyV1, %s", csiProxyV1.GetAPIVersions())
		return csiProxyV1, nil
	}
	klog.V(4).Infof("failed to connect to csi-proxy v1 with error=%v, will try with v1Beta", err)
```